src/treemap/index.js

```diff
@@ -132,10 +137,22 @@ class Treemap extends React.Component {
     return [];
   }
 
+  /**
+   * Triggers a callback on a box if the callback is set.
+   * @param {function} handler Callback function.
+   * @param {Object} d Data point of the arc.
+   * @param {Object} event Event.
+   * @private
+   */
+  _triggerHandler(handler = NOOP, d, event) {
```

Comment:
I was more thinking about using the default props so you don't need to create a new method